web/client/configs/localConfig.json

```diff
@@ -48,14 +47,24 @@
   "localizedLayerStyles": {
     "name": "mapstore_language"
   },
-  "authenticationRules": [
+  "requestsConfigurationRules": [
     {
       "urlPattern": ".*rest/geostore.*",
-      "method": "bearer"
+      "headers": {
+        "Authorization": "Bearer ${securityToken}"
+      }
     },
     {
       "urlPattern": ".*rest/config.*",
-      "method": "bearer"
+      "headers": {
+        "Authorization": "Bearer ${securityToken}"
+      }
+    },
+    {
+      "urlPattern": "http(s)?\\:\\/\\/gs-stable\\.(geo-solutions\\.it|geosolutionsgroup\\.com)(\\:443|\\:80)?\\/geoserver/.*",
+      "params": {
+        "authkey": "${securityToken}"
+      }
     }
```

Comment:
This should not be included inside the core code, it should be provided with the data
directory (we should remember to update it after merging the PR)